src/client/package.json

```diff
@@ -45,12 +45,13 @@
     "finsemble:dev": "cross-env TARGET=finsemble vite",
     "finsemble:build": "cross-env TARGET=finsemble vite build",
     "generateCod": "hydra-web-codegen -i trading-gateway.hyer -o src/generated",
-    "storybook": "start-storybook -p 6006 --no-manager-cache",
+    "storybook": "storybook dev -p 6006",
     "e2e:openfin": "playwright test --project=openfin",
     "e2e:openfin:smoke": "playwright test --project=openfin --grep @smoke",
     "e2e:web": "playwright test --project=chrome",
+    "preview": "vite preview",
     "e2e:web:smoke": "playwright test --project=chrome --grep @smoke",
-    "build-storybook": "build-storybook -o ./dist/storybook"
+    "build-storybook": "storybook build -o ./dist/storybook"
```

Comment:
should this be changed to storybook:build to match our convention?

Reply:
I agree, but i just realise from the failed build that the workflows use `build-storybook`. Not sure if that update should be in this PR

Reply:
discussed over slack to remove the change